entgql/plugin.go

```diff
@@ -64,7 +64,7 @@ func (e *gqlgenPlugin) InjectSourceEarly() *ast.Source {
 func (e *gqlgenPlugin) MutateConfig(cfg *config.Config) error {
 	if e.models != nil {
 		for name, goType := range e.models {
-			if !cfg.Models.Exists(name) {
+			if len(cfg.Models[name].Model) == 0 {
```

Comment:
This fixed issue when we use the `@goField` directive on the field.
[It creates a mapping without Models](https://github.com/99designs/gqlgen/blob/f0fdb116f45350aabf698c20bf6410283f96bb11/codegen/config/config.go#L285-L290). Then, we don't have the correct map so GQLGen will generate models for it.

It's maybe related to this issue: https://github.com/99designs/gqlgen/discussions/1993

https://github.com/cliedeman/entcontrib/blob/60b59d6c5a2656e90f6b78f9ba88645f17b60b35/entgql/plugin/config.go#L47-L49

cc @cliedeman